components/cdc/src/delegate.rs

```diff
@@ -362,12 +362,13 @@ impl Delegate {
             self.region_id,
             change_data_event,
         );
-        for i in 0..downstreams.len() - 1 {
-            if normal_only && downstreams[i].state.load() != DownstreamState::Normal {
+        for curr_downstream in downstreams.iter().take(downstreams.len() - 1) {
```

Comment:
prefer `downstream` to `curr_downstream`

Reply:
Fixed